common/autoware_perception_rviz_plugin/src/object_detection/detected_objects_display.cpp

```diff
@@ -150,9 +159,28 @@ void DetectedObjectsDisplay::processMessage(DetectedObjects::ConstSharedPtr msg)
       auto marker_ptr = yaw_rate_covariance_marker.value();
       marker_ptr->header = msg->header;
       marker_ptr->id = id++;
-      add_marker(marker_ptr);
+      // add_marker(marker_ptr);
+      markers.push_back(marker_ptr);
     }
   }
+
+  if (!markers.empty()) {
+    std::set new_marker_ids = std::set<rviz_default_plugins::displays::MarkerID>();
+    for (const auto & marker : markers) {
+      rviz_default_plugins::displays::MarkerID marker_id =
+        rviz_default_plugins::displays::MarkerID(marker->ns, marker->id);
+      add_marker(marker);
+      new_marker_ids.insert(marker_id);
+    }
+    for (auto itr = existing_marker_ids.begin(); itr != existing_marker_ids.end(); itr++) {
+      if (new_marker_ids.find(*itr) == new_marker_ids.end()) {
+        deleteMarker(*itr);
+      }
+    }
+    existing_marker_ids = new_marker_ids;
```

Comment:
In Rviz's visualization, if we add markers with the same id, RViz will trigger actions similar to"modifying" the existing markers. 
In this code, and also the code for predicted object display, we utilize this pattern to save computation. Only conduct the "delete" command when extra IDs are used.

Reply:
I check the RVIZ Common Code run again.

Running Steps:
1. add_marker will push the marker the the `m_marker_common` queue for cache
2. During the next update trigger, the plugin will add/modify markers into Rviz, and also clean up expired markers.

The current codes also work fine because all markers will be cleaned up by the automatic expired marker removal.